custom_components/wyzeapi/sensor.py

```diff
@@ -68,17 +68,23 @@ def enabled(self):
     _attr_device_class = DEVICE_CLASS_BATTERY
     _attr_native_unit_of_measurement = PERCENTAGE
 
-    # make the battery unavailable by default, this will be toggled after the first upate from the battery entity that
-    # has battery data.
-    _available = False
+    
 
     def __init__(self, lock, battery_type):
         self._enabled = None
         self._lock = lock
         self._battery_type = battery_type
+        # make the battery unavailable by default, this will be toggled after the first update from the battery entity that
+        # has battery data.
+        self._available = False
 
     @callback
     def handle_lock_update(self, lock: Lock) -> None:
+      """
```

Comment:
The indention on this block is wrong and is causing CI errors

Reply:
seems github tabs was set to "2" instead of "4" 
resolved.